chapter_convolutional-neural-networks/conv-layer.md

```diff
@@ -1,6 +1,6 @@
 # 二维卷积层
 
-卷积神经网络是含有卷积层（convolutional layer）的神经网络。本章中卷积神经网络均使用最常见的二维卷积层。它有高和宽两个空间维度，常用来处理图像数据。本节中，我们将介绍简单形式的二维卷积层的工作原理。
+卷积神经网络是含有卷积层（convolutional layer）的神经网络。本章中卷积神经网络均使用最常见的二维卷积层，它有高和宽两个空间维度，常用来处理图像数据。本节中，我们将介绍简单形式的二维卷积层的工作原理。
 
```

Comment:
We tried to keep "one sentence, one meaning". For technical discussions, short sentences are easier for readers.